Assets/MRTK/SDK/Experimental/PulseShader/Scripts/SurfacePulseOnSelect.cs

```diff
@@ -0,0 +1,204 @@
+using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEditor;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Experimental.SurfacePulse
+{
+	[AddComponentMenu("Scripts/MRTK/SDK/SurfacePulse")]
```

Comment:
nit:  missing doc comment for the class

Reply:
Added doc comment. Added to existing SurfacePulse.cs as well.